plugin/set.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/hashicorp/packer/version"
 )
 
+// Use this name to make the name of the plugin in the packer template match
+// the multiplugin suffix, instead of requirng a second part.
+const DEFAULT_NAME = "-packer-default-plugin-name-"
```

Comment:
```suggestion
// Use this name to make the name of the plugin in the packer template match
// the multiplugin suffix, instead of requiring a second part.
const DEFAULT_NAME = "-packer-default-plugin-name-"
```